x-pack/legacy/plugins/siem/public/components/ml/api/error_to_toaster.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { isError } from 'lodash/fp';
+import uuid from 'uuid';
+import { ActionToaster, AppToast } from '../../toasters';
+import { ToasterErrorsType, ToasterErrors } from './throw_if_not_ok';
+
+export type ErrorToToasterArgs = Partial<AppToast> & {
+  error: unknown;
```

Comment:
++ on the `unknown` here! And thanks for the [reading material](https://mariusschulz.com/blog/the-unknown-type-in-typescript) -- good stuff!